pennylane/io.py

```diff
@@ -146,13 +156,99 @@ def circuit_loaded_qiskit_circuit():
         return load(quantum_circuit, format="qiskit", measurements=measurements)
     except ValueError as e:
         if e.args[0].split(".")[0] == "Converter does not exist":
-            raise RuntimeError(
-                "Conversion from Qiskit requires the PennyLane-Qiskit plugin. "
-                "You can install the plugin by running: pip install pennylane-qiskit. "
-                "You may need to restart your kernel or environment after installation. "
-                "If you have any difficulties, you can reach out on the PennyLane forum at "
-                "https://discuss.pennylane.ai/c/pennylane-plugins/pennylane-qiskit/"
-            ) from e
+            raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
+        raise e
+
+
+def from_qiskit_op(qiskit_op, params=None, wires=None):
+    """Loads Qiskit SparsePauliOp objects by using the converter in the PennyLane-Qiskit plugin.
+
+    Args:
+        qiskit_op (qiskit.quantum_info.SparsePauliOp): the SparsePauliOp to be converted
+        params (Any): optional assignment of coefficient values for the SparsePauliOp; see the
+            `Qiskit documentation <https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp#assign_parameters>`__
+            to learn more about the expected format of these parameters
+        wires (Sequence | None): optional assignment of wires for the converted SparsePauliOp; if
+            the original SparsePauliOp acted on :math:`N` qubits, then this must be a sequence of
+            length :math:`N`
+
+    Returns:
+        Operator: The equivalent PennyLane operator.
+
+    .. note::
+
+        The wire ordering convention differs between PennyLane and Qiskit: PennyLane wires are
+        enumerated from left to right, while the Qiskit convention is to enumerate from right to
+        left. A ``SparsePauliOp`` term defined by the string ``"XYZ"`` applies ``Z`` on wire 0,
+        ``Y`` on wire 1, and ``X`` on wire 2.
```

Comment:
How do we know this is the case? Is it reported somewhere in the Qiskit docs?

Reply:
Passing the string "XYZ" creates a `qiskit.quantum_info.Pauli` object in Qiskit, and its described in the docs for that here: https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.Pauli

Reply:
Documentation for `SparsePauliOp` mentions that the Paulis are a `PauliList`, and documentation for the `PauliList` mentions that the elements are `Paulis`. I don't know of a place where it's explained "closer" to the SparsePauliOps class. Should we make this text more specific to describing the wire ordering of a Pauli and provide the link?

Reply:
You're right! I'm sad though because it just makes things more confusing :disappointed: 
This is how I checked:
```python
from qiskit.quantum_info import SparsePauliOp
from qiskit import QuantumCircuit
from qiskit_aer import AerSimulator
from qiskit.opflow import CircuitSampler
from qiskit.opflow import StateFn, PauliExpectation, CircuitSampler
from qiskit_aer import AerSimulator
from qiskit.opflow import CircuitStateFn, PauliSumOp
 
op = SparsePauliOp(["XZ"], [1])

state = QuantumCircuit(2)
state.h(0)

simulator = AerSimulator(method="statevector", shots=100)

# opflow stuff
opflow_op = PauliSumOp(op)
opflow_state = CircuitStateFn(state)
measurable_expression = StateFn(opflow_op, is_measurement=True).compose(opflow_state)
expectation = PauliExpectation().convert(measurable_expression)
simulator = AerSimulator(method="statevector", shots=100)
circuit_sampler = CircuitSampler(simulator).convert(expectation)
 
expectation_value = circuit_sampler.eval().real
```
The result is: `0.0`. Compared to PL:
```python
import pennylane as qml

dev = qml.device("default.qubit")

@qml.qnode(dev)
def f():
    qml.Hadamard(0)
    return qml.expval(qml.PauliX(0) @ qml.PauliZ(1)), qml.expval(qml.PauliX(1) @ qml.PauliZ(0))

f()
```
We get `(tensor(1., requires_grad=True), tensor(0., requires_grad=True))`, which agrees with `X` being on qubit `1` not `0`.

Reply:
This is mentioned in the ["String representation"](https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.Pauli) section of the documentation for `Pauli`:

> In the string representation qubit-0 corresponds to the right-most Pauli character, and qubit-($n$−1) to the left-most Pauli character. For example `'XYZ'` represents  $X$ ⊕ $X$ ⊕ $Z$ with `'Z'` on qubit-0, `'Y'` on qubit-1, and `'X'` on qubit-2.

Should I include a link to it?

Reply:
Right, yes would be nice to link to there in our warning.

Reply:
It definitely does, and we have to account for this various places in the plugin - for example, we have to reverse the order of the samples they return. We may want to think about how to document this in general for people who might be coming to PennyLane from Qiskit.

Wrt the conversation a few days ago, this is also a good reason to check the devices vs default qubit in the test suite, because this has slipped under the radar before; most recently we fixed a related issue that had been present for quite some time [in the Sampler device](https://github.com/PennyLaneAI/pennylane-qiskit/pull/357#discussion_r1421093292).